tasks/ceph.py

```diff
@@ -558,7 +558,19 @@ def cluster(ctx, config):
                         stdout=StringIO(),
                         )
 
-                remote.run(args= ['yes', run.Raw('|')] + ['sudo'] + mkfs + [dev])
+                try:
+                    remote.run(
+                        args=['yes', run.Raw('|')] + ['sudo'] + mkfs + [dev]
+                    )
+                except run.CommandFailedError:
```

Comment:
@gregsfortytwo Yeah, we discussed this in standup and I think this change needs a bit more thought.  At this point I'd be +1 on reverting the original PR that introduced this problem so your tests will finish.  We can come back later with a more elegant solution.